test/parser/parse.test.ts

```diff
@@ -1748,5 +1748,16 @@ See more help with --help`)
       })
       expect(out.flags.foo).to.equal(true)
     })
+
+    it('works with aliased short char', async () => {
+      const out = await parse(['-b'], {
+        flags: {
+          foo: Flags.boolean({
+            charAliases: ['b'],
```

Comment:
UT coverage ideas:
do a non-boolean flag
empty array
same char duplicated in aliases and charAliases
error: using the full flag and its charAliases

Reply:
I'll add these in a PR